samples/nsxt_change_vm_vif.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+"""
+Written by Yasen Simeonov
+Github: https://github.com/yasensim
+
+This code is released under the terms of the Apache 2
+http://www.apache.org/licenses/LICENSE-2.0.html
+
+Example script to change the network of the Virtual Machine NIC
+that includes NSX-T opeque switch
+
+"""
+
+import atexit
+import ssl
+
+from tools import cli
+from tools import tasks
+from pyVim import connect
+from pyVmomi import vim, vmodl
+
+
+def get_obj(content, vimtype, name):
+    """
+     Get the vsphere object associated with a given text name
+    """
+    obj = None
+    container = content.viewManager.CreateContainerView(content.rootFolder,
+                                                        vimtype, True)
+    for view in container.view:
+        if view.name == name:
+            obj = view
+            break
+    return obj
+
+
+def get_args():
+    """Get command line args from the user.
+    """
+
+    parser = cli.build_arg_parser()
+
+    parser.add_argument('-v', '--vm_name',
+                        required=True,
+                        action='store',
+                        help='Virtual machine name')
+
+    parser.add_argument('-n', '--network_name',
+                        required=True,
+                        action='store',
+                        help='Name of the portgroup or NSX-T Logical Switch')
+
+    args = parser.parse_args()
+
+    cli.prompt_for_password(args)
+    return args
+
+
+def main():
+    """
+    Simple command-line program for changing network virtual machines NIC
+    that includes NSX-T opeque switch.
+    """
+
+    args = get_args()
+    ssl._create_default_https_context = ssl._create_unverified_context
+
+    try:
+        service_instance = connect.SmartConnect(host=args.host,
```

Comment:
Provide '-S' command line option to skip cert verification instead of skipping verification by default